### PR TITLE
chore: mirror scanner-test-0.3.69 image

### PR DESCRIPTION
**Pull request by RTann**
/cc @BradLugo @daynewlee @jvdm @dcaravel @tommartensen

### CODE REVIEW

File: core-services/image-mirroring/_config.yaml [hunk 1]
```diff
@@ -555,6 +555,8 @@ supplementalCIImages:
     image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.66
   stackrox/apollo-ci:scanner-test-0.3.68:
     image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.68
+  stackrox/apollo-ci:scanner-test-0.3.69:
+    image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.69
```
Comment (jvdm) [inline core-services/image-mirroring/_config.yaml:559]: Ah, totally forgot this mirroring thing in OCI.

**Review by jvdm — APPROVED** [review]

File: core-services/image-mirroring/_config.yaml (hunk 1 above)
Comment (RTann) [reply core-services/image-mirroring/_config.yaml:559]: image

**Pull request merged**